pkg/controller/controller.go

```diff
@@ -294,7 +294,8 @@ func (c *Controller) sync(domain string) error {
 // zmsGetSignedDomains - make http request to zms API to fetch domain data
 func (c *Controller) zmsGetSignedDomains(domain string) (*zms.SignedDomains, bool, error) {
 	d := zms.DomainName(domain)
-	signedDomain, _, err := c.zmsClient.GetSignedDomains(d, "", "", "")
+	master := true
```

Comment:
Do we need to enable the master field? It seems like the field is for system principals to request data from the master data store. Ref: https://github.com/yahoo/athenz/blob/c79ce8954e14a4e2b11efc2ae0dc6bc4d66389b1/core/zms/src/main/rdl/SignedDomains.rdli#L52-L72
cc: @mcieplak if you know more about the master data store